lean4-info.el

```diff
@@ -132,14 +132,41 @@
                         'font-lock-face 'font-lock-comment-face)
             'fixedcase 'literal)))))))
 
+
+;;  debounce interval
+;; That is, we maintain the timing invariant that the time lean4-info-buffer-redisplay
+;; is invoked is at least (debounce-interval-seconds) after the last redisplay request.
```

Comment:
That logic looks kind of wrong to me; we should refresh *at most* `duration-seconds` (you used three different names btw) after any request. Otherwise with sufficiently frequent requests such as from building dependent files, we would not show anything at all for quite a while.

Reply:
@Kha Ah, I didn't know that there are cases where we get many requests. I was under the impression that a single request is generated per user movement command.

The current code "delays" rendering by `duration-seconds`, to wait for more changes that might come in. Do you suggest we add another timer (say, `debounce-upper-bound-duration`) which is a somewhat long duration such as `0.5sec`? This is so that in the case where we get a large sequence of requests, we do render *some* information once in a while.

Reply:
Hmm, do you think one refresh every 0.1s is too often? Because in that case one timer would suffice. But I'm okay with either solution.